test/unit/pipeline/show-ends.ts

```diff
@@ -0,0 +1,24 @@
+/*
+	File: test/unit/pipeline/show-ends.ts
+	cpuabuse.com
+*/
+
+/**
+ * Tests `show-ends`.
+ */
+
+/* global describe:true */
+/* global it:true */
+
+import { strictEqual } from "assert";
+import { fruitList as inputFruitList } from "../../input/pipeline";
+import { fruitList as expectedFruitList } from "../../expected/pipeline";
+import { processEnds } from "../../../src/pipeline/show-ends";

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing file extension for "../../../src/pipeline/show-ends"](https://app.codacy.com/gh/cpuabuse/cross-cat/pullRequest?prid=5058624)